From 2051ba3003372fd6e2b901b7c5606870be834f0d Mon Sep 17 00:00:00 2001
From: Eloi Bail <eloi.bail@savoirfairelinux.com>
Date: Fri, 20 Mar 2020 14:03:41 +0100
Subject: [PATCH] run_tests.sh: make it called from any directory

Get the path of run_tests.sh and change directory to its location to
be able to call it from any directory.
---
 run_tests.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/run_tests.sh b/run_tests.sh
index 01e36dc..80e473c 100755
--- a/run_tests.sh
+++ b/run_tests.sh
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+dir_path=$(dirname $(readlink -e $0))
+cd $dir_path
 verbose="no"
 tap_output="no"
 run_all_tests="no" # don't run nodefault tests
-- 
2.17.1

